SerialPrograms/Source/PokemonLA/Programs/PokemonLA_TreeActions.cpp

```diff
@@ -0,0 +1,122 @@
+/*  Tree Actions
+ *
+ *  From: https://github.com/PokemonAutomation/Arduino-Source
+ *
+ */
+
+#include "CommonFramework/VideoPipeline/VideoFeed.h"
+#include "CommonFramework/InferenceInfra/InferenceRoutines.h"
+#include "PokemonLA/Programs/PokemonLA_MountChange.h"
+#include "PokemonLA/Inference/Battles/PokemonLA_BattleMenuDetector.h"
+#include "PokemonLA_RegionNavigation.h"
+#include "PokemonLA_TreeActions.h"
+#include "PokemonLA/Resources/PokemonLA_PokemonInfo.h"
+#include "PokemonLA/Inference/PokemonLA_StatusInfoScreenDetector.h"
+#include "Pokemon/Inference/Pokemon_NameReader.h"
+
+namespace PokemonAutomation{
+namespace NintendoSwitch{
+namespace PokemonLA{
+
+
+void setup(ProgramEnvironment& env, ConsoleHandle& console, BotBaseContext& context){
+    goto_camp_from_jubilife(env, console, context, TravelLocations::instance().Fieldlands_Heights);
+    pbf_move_left_joystick(context, 170, 255, 30, 30);
+    change_mount(console, context, MountState::BRAVIARY_ON);
+    pbf_press_button(context, BUTTON_B, (6.35 * TICKS_PER_SECOND), 20);
+    pbf_press_button(context, BUTTON_PLUS, 20, 20);
+    pbf_wait(context, (1 * TICKS_PER_SECOND));
```

Comment:
If you want, you can merge each of these waits into the button press above. The last parameter of `pbf_press_button` and family is the wait.

Reply:
removed the waits here and in the BurmyFinder too :) thanks!